Orange/widgets/tests/base.py

```diff
@@ -225,6 +225,36 @@ def send_signal(self, input, value, *args, widget=None, wait=-1):
         """
         if widget is None:
             widget = self.widget
+        self._send_signal(widget, input, value, *args)
+        widget.handleNewSignals()
+        if wait >= 0 and widget.isBlocking():
+            spy = QSignalSpy(widget.blockingStateChanged)
+            self.assertTrue(spy.wait(timeout=wait))
+
+    def send_signals(self, inputs, values, *args, widget=None, wait=-1):
```

Comment:
One more thing: I think a better signature would be `def send_signals(self, signals, *args, widget=None, wait=-1):` where `signals` is a list of tuples. Otherwise the caller has to correctly pair elements of two lists, and the method is then only simpler (it just avoids a `zip`.